firebase-firestore/ktx/src/test/kotlin/com/google/firebase/firestore/ktx/UnitTestUtilTests.kt

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 Google LLC
```

Comment:
Code in testutil seems trivial enough that this test is not necessary.

Reply:
Thank you, I deleted this file. I was not sure if I should unit test test utilities before.